### PR TITLE
Make next-sitemap use a LABELFLOW_URL environment variable

### LINKED ISSUE #789: Make next-sitemap use a LABELFLOW_URL environment variable

### PR DESCRIPTION
**Pull request by Jordanlelay**
## Work performed

- Update sitemap config changing an env variable

## Resolved issues

Closes #789

### CODE REVIEW

File: typescript/web/next-sitemap.js [hunk 1]
```diff
@@ -1,15 +1,5 @@
-const { VERCEL_URL } = process.env;
-
-if (!VERCEL_URL) {
-  throw new Error("Missing required environment variable VERCEL_URL");
-}
-
-const SITE_URL = VERCEL_URL.includes("://")
-  ? VERCEL_URL
-  : `https://${VERCEL_URL}`;
-
```
Comment (bbenoist) [inline typescript/web/next-sitemap.js:10]: @Jordanlelay Why have you removed those checks?

Comment (Jordanlelay) [reply typescript/web/next-sitemap.js:10]: Because I won't be using them as I use a dedicated variable

Comment (bbenoist) [reply typescript/web/next-sitemap.js:10]: The lines 3 to 5 are still useful.
But you're right for the lines 7 to 9 👍

Comment (Jordanlelay) [reply typescript/web/next-sitemap.js]: I added it back

Comment (bbenoist) [reply typescript/web/next-sitemap.js]: Yes, but you need to replace VERCEL_URL by LABELFLOW_URL in the check now 😉

Comment (Jordanlelay) [reply typescript/web/next-sitemap.js]: 🤦  fixed

**Review by bbenoist — APPROVED** [review]

**Pull request merged**